packages/mobile/src/components/track-list/TrackListItem.tsx

```diff
@@ -0,0 +1,313 @@
+  isLoading: boolean
+  isPlaying: boolean
+}
+
+const useStyles = makeStyles(({ palette, spacing }) => ({
+  artworkContainer: {
+    position: 'relative',
+    flexGrow: 0,
+    flexShrink: 0,
+    flexBasis: 52,
+    height: 52,
+    width: 52,
+    marginRight: spacing(4),
+    borderRadius: 4
+  },
+  artwork: {
+    height: '100%',
+    width: '100%',
+    borderRadius: 4,
+    justifyContent: 'center',
+    alignItems: 'center'
+  },
+  artworkIcon: {
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    right: 0,
+    bottom: 0,
+    backgroundColor: 'rgba(0,0,0,0.4)',
+    borderRadius: 4
+  },
+  artworkIconSvg: {
+    position: 'absolute',
+    top: '50%',
+    left: '50%',
+    height: '40%',
+    width: '40%'
+  },
+  trackContainer: {
+    width: '100%',
+    height: 72,
+    backgroundColor: palette.white
+  },
+  trackContainerActive: {
+    backgroundColor: palette.neutralLight9
+  },
+  trackInnerContainer: {
+    height: '100%',
+    width: '100%',
+    flexDirection: 'row',
+    justifyContent: 'flex-start',
+    alignItems: 'center',
+    paddingVertical: spacing(3),
+    paddingHorizontal: spacing(6)
+  },
+  nameArtistContainer: {
+    flex: 1,
+    justifyContent: 'center',
+    overflow: 'hidden',
+    height: '100%'
+  },
+  trackTitle: {
+    ...font('demiBold'),
+    color: palette.neutral
+  },
+  artistName: {
+    ...font('medium'),
+    color: palette.neutralLight2,
+    alignItems: 'center'
+  },
+  iconContainer: {
+    marginLeft: spacing(2)
+  },
+  playButtonContainer: {
+    marginRight: spacing(4)
+  }
+}))
+
+const ArtworkIcon = ({ isLoading, isPlaying }: ArtworkIconProps) => {
+  const [artworkHeight, setArtworkHeight] = useState(0)
+  const [artworkWidth, setArtworkWidth] = useState(0)
+  const styles = useStyles()
+  let artworkIcon
+
+  const handleLayout = e => {
+    setArtworkWidth(e.nativeEvent.layout.width)
+    setArtworkHeight(e.nativeEvent.layout.height)
+  }
+
+  if (isLoading) {
+    artworkIcon = (
+      // <div className={styles.loadingAnimation}>
+      //   <Lottie
+      //     options={{
+      //       loop: true,
+      //       autoplay: true,
+      //       animationData: loadingSpinner
+      //     }}
+      //   />
+      // </div>
+      <View>
+        <Text>Loading...</Text>
+      </View>
+    )
+  } else {
+    const Icon = isPlaying ? IconPause : IconPlay
+    artworkIcon = (
+      <Icon
+        style={[
+          styles.artworkIconSvg,
+          {
+            transform: [
+              {
+                translateX: -(artworkWidth / 2),
+                translateY: -(artworkHeight / 2)
+              }
+            ]
+          }
+        ]}
+      />
+    )
+  }
+
+  return (
+    <View onLayout={handleLayout} style={styles.artworkIcon}>
+      {artworkIcon}
+    </View>
+  )
+}
+
+type ArtworkProps = {
+  trackId: ID
+  isLoading: boolean
+  isActive?: boolean
+  isPlaying: boolean
+  coverArtSizes: CoverArtSizes
+}
+
+const Artwork = ({
```

Comment:
would love to know if we have a component similar to this that we can share, some kinda `TrackCoverArt` component?